### PR TITLE
designed avatars-badges

### PR DESCRIPTION
**Pull request by itsmebipul00**
Designed Avatars and Badges

### CODE REVIEW

File: Components/Avatars/Avatars.css [hunk 1]
```diff
@@ -0,0 +1,172 @@
...
+  background: hsl(var(--clr-info));
+}
+
+.notification-color-yellow::after {
+  background: hsl(var(--clr-warning));
+}
+
+.width-5rem {
+  width: 5rem;
+}
```
Comment (dev-prabhat) [inline Components/Avatars/Avatars.css:86]: you can use width-sm sm= small and same with others also like lg , xl

File: Components/Avatars/Avatars.css [hunk 2]
```diff
@@ -0,0 +1,172 @@
...
+.avatar {
+  display: flex;
+  gap: 5rem;
+  margin: 3rem;
+  width: 80%;
+  align-items: flex-end;
+}
+
+.radius-50 {
+  border-radius: 50%;
```
Comment (dev-prabhat) [inline Components/Avatars/Avatars.css:14]: same with radius like sm , md , lg and xl

File: Components/Avatars/Avatars.css [hunk 3]
```diff
@@ -0,0 +1,172 @@
...
+  aspect-ratio: 1;
+}
+
+.placeholder-face {
+  height: 40%;
+  aspect-ratio: 1;
+  border: 0.1875rem solid hsl(var(--clr-dark));
+  border-radius: 50%;
+  background: hsl(var(--clr-white));
+  z-index: 999;
```
Comment (zoef-yt) [inline Components/Avatars/Avatars.css:28]: change the z-index

File: Components/Avatars/Avatars.css [hunk 4]
```diff
@@ -0,0 +1,172 @@
...
+  grid-area: code-icon-green;
+}
+
+.code-icon-green,
+.code-icon-blue,
+.code-icon-yellow {
+  margin-right: auto;
+  width: 90%;
+  padding: 1rem !important;
+  /* height: 10vh; */
```
Comment (zoef-yt) [inline Components/Avatars/Avatars.css:160]: remove this comment

**Review by zoef-yt — CHANGES_REQUESTED** [review]

**Review by dev-prabhat — CHANGES_REQUESTED** [review]
Naming can be change

File: Components/Avatars/Avatars.css (hunk 4 above)
Comment (iRohitGaur) [reply Components/Avatars/Avatars.css:160]: +1

**Review by iRohitGaur — APPROVED** [review]
Suggestion: I feel the code is too much modular to a level where you will have to use a lot of classes in your html elements to get things working. Think about it once.

Other than that, great work. Good to merge 👍🏻

File: Components/Avatars/Avatars.css (hunk 1 above)
Comment (bhushannaikaroskar) [reply Components/Avatars/Avatars.css:86]: +1 you can use conventions like width-sm, width-md : medium width and so on. later you can also alter the values this way.

File: Components/Avatars/Avatars.css [hunk 5]
```diff
@@ -0,0 +1,172 @@
...
+  grid-area: code-icon-yellow;
+}
+
+.notification-color-yellow {
+  grid-area: notification-color-yellow;
+}
+
+.notification-color-green {
+  grid-area: notification-color-green;
+}
```
Comment (bhushannaikaroskar) [inline Components/Avatars/Avatars.css:144]: I would suggest to use names like notification-color-success or notification-success rather that defining values such as .notification-green but still if you sure thinks are not gonna change in future keep it that way.

**Review by bhushannaikaroskar — APPROVED** [review]

File: Components/Avatars/Avatars.css (hunk 3 above)
Comment (itsmebipul00) [reply Components/Avatars/Avatars.css:28]: this is there for the placeholder face to stack above placeholder hands

File: Components/Avatars/Avatars.css (hunk 4 above)
Comment (itsmebipul00) [reply Components/Avatars/Avatars.css:160]: noted

File: Components/Avatars/Avatars.css (hunk 1 above)
Comment (itsmebipul00) [reply Components/Avatars/Avatars.css:86]: noted

File: Components/Avatars/Avatars.css (hunk 2 above)
Comment (itsmebipul00) [reply Components/Avatars/Avatars.css:14]: noted

File: Components/Avatars/Avatars.css (hunk 5 above)
Comment (itsmebipul00) [reply Components/Avatars/Avatars.css:144]: noted

**Pull request merged**